docs/website/src/landing/business-value/business-value.less

```diff
@@ -18,6 +15,47 @@
       padding: 0.5rem;
       font-size: 14px;
     }
+
+    &.design-1 {
+      background-color: @landing-blue;
+      color: #fff;
```

Comment:
If i'm not mistaken, there are variables:
@landing-bg-color: #fff;
@landing-fg-color: #000;